packages/inter-protocol/src/vaultFactory/liquidation.js

```diff
@@ -47,6 +51,12 @@ const scheduleLiquidationWakeups = async (
     E(auctioneerPublicFacet).getGovernedParams(),
     E(timer).getCurrentTimestamp(),
   ]);
+  if (lastCurrentTime) {
+    if (TimeMath.compareAbs(now, lastCurrentTime) <= 0) {
+      Fail`time must advance`;
```

Comment:
_question_:
what will happen to auction state if this `Fail` fires?

Reply:
It's a sanity check. Adding it detected the infinite loop we were hitting in debugging. 

> I'm not sure what this clause is here to protect us from

I think it's safe to assume time won't move backwards. We were seeing wakeups at the same time over and over. This guarded against that but I agree it's overly defensive now.

I'll remove it.